ui/components/ui/typography/README.mdx

```diff
@@ -0,0 +1,16 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs';
+import List from '.';
```

Comment:
`Typography` is the component we want to document here and what we should be using for props in the `ArgsTable`. List is the storybook component and only used for the stories.

```suggestion
import Typography from '.';
```